mantidproject/mantid
# ISIS SANS: Provide diagnostic output, ie put the transmission workspaces

The transmission workspaces are sometimes used to diagnose issues with a reduction. This pull request adds an option to the sans GUI v2 to output these workspaces if required. 

**To test:**

- Get the data set from \\olympic\Babylon5\Scratch\Matthew Andrew\GUI_TEST
- Open up the sans GUI v2 in mantid
- Load the user file _USER_SANS2D_154E_2p4_4m_M3_Xpress_8mm_SampleChanger_REAR_ from the folder
- Load the _batch\_ file_ from the folder
- Click process and verify that only one work space is produced. 
- Navigate to the settings->adjustment tab and tick the box entitled show transmission.
- Rerun the reduction and check that the transmission and unfitted transmission work spaces are produced. 




Fixes #21017 . 

**Release Notes** 
Have added information to the release notes and the docs.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
